extensions/amp-analytics/0.1/events.js

```diff
@@ -1494,31 +1521,28 @@ export class VisibilityTracker extends EventTracker {
       // false missed searches.
       const selectionMethod =
         config['selectionMethod'] || visibilitySpec['selectionMethod'];
-      unlistenPromise = this.root
-        .getAmpElement(
-          context.parentElement || context,
+
+      if (multiSelectorVisibilityOn && Array.isArray(selector)) {
+        unlistenPromise = Promise.all(
```

Comment:
Can you do 
```
selector = isArray(selector) ? selector : [selector]
```
first

Reply:
Thinking ahead, we will still have to keep array code path and non-array codepath separate because we will be applying querySelectorAll to only the array path.

Reply:
Good point. nvm